signingscript/script.py

```diff
@@ -56,35 +64,46 @@ def write_json(self, *args):
 
 
 # config {{{1
-def get_default_config():
-    """ Create the default config to work from.
+def get_default_config(base_dir=None):
+    """Create the default config to work from.
+
+    Args:
+        base_dir (str, optional): the directory above the `work_dir` and `artifact_dir`.
+            If None, use `..`  Defaults to None.
+
+    Returns:
+        dict: the default configuration dict.
     """
-    cwd = os.getcwd()
-    parent_dir = os.path.dirname(cwd)
+    base_dir = base_dir or os.path.dirname(os.getcwd())
     default_config = {
         'signing_server_config': 'server_config.json',
-        'tools_dir': os.path.join(parent_dir, 'build-tools'),
-        'work_dir': os.path.join(parent_dir, 'work_dir'),
-        'artifact_dir': os.path.join(parent_dir, '/src/signing/artifact_dir'),
+        'work_dir': os.path.join(base_dir, 'work_dir'),
+        'artifact_dir': os.path.join(base_dir, '/src/signing/artifact_dir'),
         'my_ip': "127.0.0.1",
         'token_duration_seconds': 20 * 60,
         'ssl_cert': None,
         'signtool': "signtool",
-        'schema_file': os.path.join(cwd, 'signingscript', 'data', 'signing_task_schema.json'),
+        'schema_file': os.path.join(os.path.dirname(__file__), 'data', 'signing_task_schema.json'),
         'verbose': True,
+        'zipalign': 'zipalign',
```

Comment:
I remember not adding zipalign, because it's usually not in $PATH. I should have added an inline comment. Do you think it's worth adding this default value?